docs/docs/constraints.md

```diff
@@ -4,31 +4,31 @@ title: Constraints
 
 # Constraints
 
-Constraints control where apps run to allow optimizing for fault tolerance or locality. They are made up of three parts: a field name, an operator, and an optional parameter. The field can be the slave hostname or any Mesos slave attribute.
+Constraints control where apps run to enable you to optimize for fault tolerance or location. Constraints have three parts: a field name, an operator, and an optional parameter. The field can be the slave hostname or any Mesos slave attribute.
 
 ## Fields
 
 ### Hostname field
 
-`hostname` field matches the slave hostnames, see `UNIQUE operator` for usage example.
+The `hostname` field matches the slave hostnames. See `UNIQUE operator`, below, for a usage example.
```

Comment:
Is "hostnames" supposed to be plural here? Or does the field match only one slave hostname?